index.js

```diff
@@ -164,6 +168,7 @@ Router.prototype.handle = function handle(req, res, callback) {
   var self = this
   var slashAdded = false
   var paramcalled = {}
+  var events = self._events
```

Comment:
Let's not peak into the internals of Node.js if it's not required, and I see no reason this is required. If there is, please enlighten me :)